pytrip/__init__.py

```diff
@@ -24,6 +24,7 @@
 # flake8: noqa
 
 import logging
+from pytrip.cube import Cube
```

Comment:
What about using numpy.save to save numpy cube (portable !) and some JSON serialiser to rest of class attributes (header-equivalent) ? This would probably mean two files :/